src/lib/libwebaudio.js

```diff
@@ -294,8 +300,8 @@ let LibraryWebAudio = {
       numberOfOutputs: HEAP32[options+1],
       outputChannelCount: HEAPU32[options+2] ? readChannelCountArray(HEAPU32[options+2]>>2, HEAP32[options+1]) : void 0,
       processorOptions: {
-        'cb': callback,
-        'ud': userData,
+        'cb': {{{ toIndexType('callback') }}},
```

Comment:
Can you do this instead in audio_worklet at the point of ` Module['wasmTable'].get`?